lib/ansible/module_utils/urls.py

```diff
@@ -606,11 +606,11 @@ def get_ca_certs(self):
                     full_path = os.path.join(path, f)
                     if os.path.isfile(full_path) and os.path.splitext(f)[1] in ('.crt','.pem'):
                         try:
-                            cert_file = open(full_path, 'r')
+                            cert_file = open(full_path, 'br')
                             os.write(tmp_fd, cert_file.read())
-                            os.write(tmp_fd, '\n')
+                            os.write(tmp_fd, b'\n')
```

Comment:
Because this is in a module_utils file that could be run on python2.4 we need to use the helper functions from six for this:
``` python

from ansible.module_utils.six import b
[...]
os.write(tmp-fd, b('\n'))
```